wrangler.toml

```diff
@@ -1,5 +1,6 @@
 main = "./dist/worker.mjs"
-compatibility_date = "2023-04-01"
+compatibility_date = "2025-03-10"
+compatibility_flags = [ "nodejs_compat" ]
```

Comment:
do you know why this is required? from a crypto standpoint, I wonder how much that changes for instance, and consider it risky. Ideally we keep it disabled